package.json

```diff
@@ -25,15 +26,18 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "dependencies": {
-    "@webex/components": "^1.18.1",
+    "@webex/component-adapter-interfaces": "^1.2.1",
```

Comment:
Can you have the install/remove from components repot to interface repo on its own commit?